README.md

```diff
@@ -87,13 +92,13 @@ This would transform these example StatsD metrics into Prometheus metrics as
 follows:
 
     test.dispatcher.FooProcessor.send.success (counter)
```

Comment:
I think you can get rid of the "(counter)" and "(gauge)" comments now that they don't actually have an effect in the example anymore.